gr-uhd/grc/uhd_fpga_fosphor.block.yml

```diff
@@ -0,0 +1,38 @@
+id: uhd_fpga_f15
+label: RFNoC F15 (FPGA Fosphor)
+
+parameters:
+-   id: type
+    label: RFNoC Block Type
+    dtype: enum
+    default: 'block'
+    options: ['block', 'sep', 'device']
+    hide: all
+-   id: desc
+    label: Block Descriptor
+    dtype: string
+    default: 'fosphor.yml'
+    hide: all
+
+inputs:
+-   domain: rfnoc.data
+    id: fft_in
+    dtype: fc32
+    multiplicity: ${ NUM_PORTS }
+
+outputs:
+-   domain: rfnoc.data
+    id: hist
+    dtype: fc32
+    multiplicity: ${ NUM_PORTS }
+-   domain: rfnoc.data
+    id: wf
+    dtype: fc32
+    multiplicity: ${ NUM_PORTS }
+
+documentation: |-
+  RFNoC F15 (fosphor) (FPGA Implementation).
```

Comment:
you'll have to explain what the *F15* stands for, somewhere :D

Reply:
Honestly, I don't know ☹️ . Sylvain started calling fosphor-in-rfnoc that as a shorthand, and I know we talked about it, but if we did I can't remember. Maybe it's short for fosphor-over-rfnoc (f + 15 letters).

Reply:
@smunaut can you chime in?

Reply:
15 is the atomic number for Phosphorus

For me it's fine to just use "RFNoC fosphor".  My main concern was in another place where it was just called "fosphor" without the "RFNoC" denomination and that is a different thing.

Reply:
Ah that's right! I know remember the conversation. Thanks for the reminder!

And yes, I'll be careful to distinguish this from regular fosphor wherever appropriate.